_references.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference path="./typings/index.d.ts" />
```

Comment:
This one I am less confident about, but I know a lot of editors will use the `_references.d.ts` file as the source of all definition files. This just specifies the main `typings` file which then loads each of the downloaded dependencies.